src/gui/uithememanager.cpp

```diff
@@ -30,17 +30,489 @@
+        {"TransferFilter.Tracker.PauseAction", "media-playback-pause"},
+        {"TransferFilter.Tracker.RemoveAction", "edit-delete"},
+        {"TransferFilter.Tracker.ResumeAction", "media-playback-start"},
+        {"TransferFilter.WithoutTrackers", "network-server"},
+        {"TransferList.BottomAction", "go-bottom"},
+        {"TransferList.Category", "view-categories"},
+        {"TransferList.Category.AddAction", "list-add"},
+        {"TransferList.Category.ResetAction", "edit-clear"},
+        {"TransferList.Category.SavePath", "inode-directory"},
+        {"TransferList.CopyAction", "edit-copy"},
+        {"TransferList.DestinationFolder.OpenAction", "inode-directory"},
+        {"TransferList.DownAction", "go-down"},
+        {"TransferList.DownloadLimit", "kt-set-max-download-speed"},
+        {"TransferList.ForceReannounce", "document-edit-verify"},
+        {"TransferList.ForceRecheck", "document-edit-verify"},
+        {"TransferList.ForceResumeAction", "media-seek-forward"},
+        {"TransferList.Hash.CopyAction", "edit-copy"},
+        {"TransferList.MagnetLink.CopyAction", "kt-magnet"},
+        {"TransferList.MaxRatio", "ratio"},
+        {"TransferList.Name.CopyActin", "edit-copy"},
+        {"TransferList.PauseAction", "media-playback-pause"},
+        {"TransferList.PreviewAction", "view-preview"},
+        {"TransferList.RemoveAction", "edit-delete"},
+        {"TransferList.RenameAction", "edit-rename"},
+        {"TransferList.ResumeAction", "media-playback-start"},
+        {"TransferList.Tackers.RenameAction", "edit-rename"},
+        {"TransferList.Tags", "view-categories"},
+        {"TransferList.Tags.AddAction", "list-add"},
+        {"TransferList.Tags.RemoveAllAction", "edit-clear"},
+        {"TransferList.TopAction", "go-top"},
+        {"TransferList.TorrentSaveLocation", "inode-directory"},
+        {"TransferList.UpAction", "go-up"},
+        {"TransferList.UploadLimit", "kt-set-max-upload-speed"},
+        {"Tray.DarkLogo", "qbittorrent-tray-dark"}, {"Tray.Download", "download"},
+        {"Tray.LightLogo", "qbittorrent-tray-light"},
+        {"Tray.Logo", "qbittorrent-tray"}, {"Tray.Seeding", "seeding"},
+        {"WebSeed.AddAction", "list-add"}, {"WebSeed.RemoveAction", "list-remove"},
+        {"WebSeedURL.CopyAction", "edit-copy"},
+        {"WebSeedURL.EditAction", "edit-rename"} };
+#endif
+}
 
 UIThemeManager *UIThemeManager::m_instance = nullptr;
 
+namespace
+{
+    class ThemeError : public RuntimeError
+    {
+    public:
+        using RuntimeError::RuntimeError;
+    };
+
+    /*
+        Loads Json Icon Config file from `configFile`, and transforms it into a map
+
+        function throws ThemeError if encountered with any error
+     */
+    QHash<QString, QString> loadIconConfig(const QString &configFile)
+    {
+        // load icon config from file
+        QFile iconJsonMap(configFile);
+        if (!iconJsonMap.open(QIODevice::ReadOnly))
```

Comment:
actually uncrustify removes it automatically.

Reply:
Then uncrustify misconfigured. You should follow qBittorrent Coding Guidelines.

Reply:
I have a habit of regularly formatting the file even while writing so it will be cumbersome to do it manually since on next format they're gone. So Can I convert these in single liners instead?

Reply:
>So Can I convert these in single liners instead?

If the lines don't get too long.
Otherwise you can try to customize uncrustify config to follow our coding style more fully.